.github/workflows/nvmx.yml

```diff
@@ -0,0 +1,19 @@
+name: NVMx testing
+on: push
```

Comment:
Do you want to run it on push to every branch. What about something like

```suggestion
on: 
  push:
    branches:
      - cas-534-nvmx
```

Reply:
Can't hurt to filter somewhat :+1: